db/migrate/20220401161223_add_refs_to_comments.rb

```diff
@@ -0,0 +1,9 @@
+class AddRefsToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+    add_reference :comments, :post, null: false, foreign_key: true
+    rename_column :comments, :user_id, :author_id
+    add_index :comments, :user
+    add_index :comments, :post
```

Comment:
- [ ] **[OPTIONAL]** I suggest generating a new migration file and using the newly generated migration to rename your table column instead of adding it here.

Reply:
Ok. Thank you for the suggestions. I will do.